### PR TITLE
fix vsearch cluster filtering

### PR DESCRIPTION
**Pull request by d4straub**
Closes https://github.com/nf-core/ampliseq/issues/696

I have not confirmed myself whether that change allows for more clusters to pass.

My python skills are horrible, so I need some serious opinion whether that seems fine. Tests pass and test files seem fine as well.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,11 +15,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Fixed`
 
 - [#697](https://github.com/nf-core/ampliseq/pull/697),[#699](https://github.com/nf-core/ampliseq/pull/699),[#713](https://github.com/nf-core/ampliseq/pull/713) - Template update for nf-core/tools version 2.13.1
+- [#711](https://github.com/nf-core/ampliseq/pull/711) - From r207 and onwards Archaea sequences were omitted when parsing GTDB databases
```
Comment (erikrikarddaniel) [inline CHANGELOG.md:18]:
```suggestion
- [#711](https://github.com/nf-core/ampliseq/pull/711) - From r207 and onwards Archaea sequences were omitted when parsing GTDB databases. (This did not affect `sbdi-gtdb` databases, only `gtdb`.)
```

**Review by erikrikarddaniel — COMMENTED** [review]
As I mentioned on Slack, I'm not competent here either. But by accident I saw a message that could be made clearer, so made a suggestion.

**Review by a4000 — APPROVED** [review]
The changes look good to me

**Comment by d4straub** [comment]
Thanks for the reviews! I am currently testing the test_full dataset with the change and hope it passes (with --vsearch_cluster and without the proposed fix it failed).

**Pull request merged**